workflow/scripts/prioritization/prediction.py

```
@@ -244,126 +251,174 @@ def extract_epilens(lengths):
+        """Split a FASTA file into smaller batch files of at most batch_size
+        sequences each.  Returns a list of batch file paths.  Original
+        sequence IDs (the >N header lines) are preserved so results can be
+        merged back without remapping."""
+        batches = []
+        current_lines = []
+        seq_count = 0
+
+        with open(fa_file, 'r') as fh:
+            for line in fh:
+                if line.startswith('>'):
+                    # start of a new sequence – flush if batch is full
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)